cmds/cmd_moderation.py

```diff
@@ -599,7 +661,16 @@ async def grab_guild_message(message, args, client, **kwargs):
         'permission': 'moderator',
         'cache': 'keep',
         'execute': grab_guild_message
-        }
+        },
+    'purge':
+        {
+            'pretty_name': 'purge <limit> [user]',
+            'description': 'Purge messages from a given channel and optionally only from a specified user',
+            'rich_description': 'Can only purge up to 100 messages at a time to prevent catastrophic errors',
```

Comment:
this or point out this is a discord limitation, also maybe add that this can only delete 14 day or younger messages, although that another check and that's gonna be some kind of discord error, I think HTTPException, although the lib may handle it.

Reply:
eh if people dont know how purge works thats their fault not mine